src/ortho/mcode/memsegs_c.c

```diff
@@ -28,7 +28,7 @@
    set rights.
 */
 
-#ifdef __APPLE__
+#if defined(__APPLE__) || !defined(MREMAP_MAYMOVE)
```

Comment:
If freebsd supports mremap(2), better to use it.
Simply add #define MREMAP_MAYMOVE 0